### DISCUSSION TITLE
post/content5/

### DISCUSSION THREAD

**Discussion by giscus** [srn221B/blog#1] [Comments]
# post/content5/

！？

http://localhost:1313/post/content5/

**Comment by srn221B** [comment]
test

**Comment by srn221B** [comment]
そのほか更新メモはここに記録していく

**Reply by srn221B** [reply]
[20220109]updated
- コメント機能の追加
   - 参考
     - https://zhauniarovich.com/post/2021/2021-06-giscus/
     - https://github.com/giscus/giscus 
- posts->archive
     - https://adityatelange.github.io/hugo-PaperMod/
